src/features/commands.ts

```diff
@@ -54,7 +54,9 @@ export default function registerCommands(server: OmniSharpServer, eventStream: E
     return new CompositeDisposable(d1, d2, d3, d4, d5, d6, d7, d8, d9, d10);
 }
 
-function restartOmniSharp(server: OmniSharpServer) {
+function restartOmniSharp(server: OmniSharpServer, eventStream: EventStream) {
+    eventStream.post(new ShowOmniSharpChannel());
+    //show the channel when restarting omnisharp
```

Comment:
Shouldn't deciding whether to show the channel be a feature of the server component?